projects/core/src/user/store/reducers/customer-coupon.reducer.ts

```diff
@@ -0,0 +1,39 @@
+import * as fromActions from '../actions/customer-coupon.action';
```

Comment:
This file need to be formatted. You can run script "prettier-fix" in package.json, this will help you to format all files

Reply:
updated